### PR TITLE
Adding more EDU videos

### PR DESCRIPTION
**Pull request by ravindk89**
Request from @petehnath to add some of our newer EDU videos to the docs.

Still need to finish work on a YAML-based setup that does not require as much manual work.

Staged:

- http://192.241.195.202:9000/staging/EDU-VIDEOS/linux/index.html

### CODE REVIEW

File: source/includes/container/installation.rst [hunk 1]
```diff
@@ -10,6 +10,13 @@ Install and Deploy MinIO
    :local:
    :depth: 1
 
+.. container:: extlinks-video
+
+   - `Installing and Running MinIO on Docker: Docker Compose Overview <https://youtu.be/FtJA3TmjaJQ?ref=docs>`__
+   - `Installing and Running MinIO on Docker: Docker Compose Lab: <https://youtu.be/tRlEctAwkk8?ref=docs>`__
+   - `Installing and Running MinIO on Docker: Overview <https://youtu.be/mg9NRR6Js1s?ref=docs>`__
+   - `Installing and Running MinIO on Docker: Installation Lab <https://youtu.be/Z0FtabDUPtU?ref=docs>`__
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/includes/container/installation.rst:18]: Eh, it's not so terrible seeing it staged, I guess. But the above might still make it better/easier to consume.

**Review by djwfyi — APPROVED** [review]
LGTM

One suggestion to consider for the page that has four videos with the same long, `:`-ed title.

File: source/includes/container/installation.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (ravindk89) [reply source/includes/container/installation.rst:18]: I like your suggestion - lemme try a few tweaks.

**Review by feorlen — APPROVED** [review]
👍🏻

**Comment by ravindk89** [comment]
@djwfyi restaged with a minor fixup

**Review by djwfyi — APPROVED** [review]
LGTM

**Pull request merged**